src/main/kotlin/com/example/demo/app/controller/PostController.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.demo.app.controller
+
+
+import com.example.demo.domain.entity.Post
+import com.example.demo.domain.repository.PostRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.*
+import javax.persistence.EntityManager
+import javax.persistence.PersistenceContext
+
+@Controller
+@RequestMapping(path = ["/post"])
+class PostController {
+    @Autowired
+    lateinit var postRepository: PostRepository
+
+    @PersistenceContext
+    private lateinit var entityManager: EntityManager
+
+    @PostMapping
+    @ResponseBody
+    fun addPost(
+        @RequestParam userId: Int, @RequestParam text: String
```

Comment:
[imo]
改行入れてある方が読みやすいと思います。
アノテーションが行の途中から始まるのが若干読みにくいな〜と。
(好みの範疇なので修正してほしいというわけではないです)

例: 
```suggestion
        @RequestParam userId: Int,
        @RequestParam text: String
```

Reply:
改行してある方が綺麗ですね。
修正しておきます。